dppl/sycl_core.pyx

```diff
@@ -34,104 +34,163 @@ from cpython.pycapsule cimport (PyCapsule_New,
                                 PyCapsule_IsValid,
                                 PyCapsule_GetPointer)
 from enum import Enum, auto
+import logging
+
+_logger = logging.getLogger(__name__)
 
 class device_type(Enum):
     gpu = auto()
     cpu = auto()
 
 
 cdef class UnsupportedDeviceTypeError(Exception):
-    """This exception is raised when a device type other than CPU or GPU is
+    '''This exception is raised when a device type other than CPU or GPU is
        encountered.
-    """
+    '''
     pass
 
 
-cdef extern from "dppl_sycl_interface.hpp" namespace "dppl":
+cdef extern from "dppl_sycl_queue_interface.hpp" namespace "dppl":
```

Comment:
This header file is deleted in the latest commit. 
I presume this needs to be converted to using the C-API?

Reply:
Done in 255b29d